cves/CVE-2017-14056.yml

```diff
@@ -125,7 +136,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The code specific to the vulnerability was not changed much throughout
+    its existence. The lines specific to it where only changed once, where a function's
+    name was changed. The vulnerability did exist for over 9 years, so many small 
+    refactorings are throughout the file that were applied to other similar file 
+    types, but overall not much changed. 
```

Comment:
Great work discovering how long this vulnerability existed. It's surprising that over the span of 9 years this was never caught, I wonder how many times the system became unstable and the developers assumed the cause was another issue.